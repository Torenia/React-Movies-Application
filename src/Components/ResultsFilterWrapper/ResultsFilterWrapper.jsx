import styled from 'styled-components';

export const ResultsFilterWrapper = styled.ul`
  padding: 20px 0;
  display: flex;
  list-style-type: none;
  margin-left: -10px;
  cursor: pointer;
  text-transform: uppercase;
  font-size: 15px;
  
  li {
    color: #fff;
    padding: 0 10px;
  }
`;
