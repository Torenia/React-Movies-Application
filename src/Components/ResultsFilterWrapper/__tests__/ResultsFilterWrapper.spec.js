import React from 'react';

import { ResultsFilterWrapper } from '../ResultsFilterWrapper';

describe('<ResultsFilterWrapper />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<ResultsFilterWrapper />);
  });

  it('should render result filter wrapper', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
