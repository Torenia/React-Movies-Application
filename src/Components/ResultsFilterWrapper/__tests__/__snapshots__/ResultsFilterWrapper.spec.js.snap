// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ResultsFilterWrapper /> should render result filter wrapper 1`] = `
.c0 {
  padding: 20px 0;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  list-style-type: none;
  margin-left: -10px;
  cursor: pointer;
  text-transform: uppercase;
  font-size: 15px;
}

.c0 li {
  color: #fff;
  padding: 0 10px;
}

<ul
  className="c0"
/>
`;
